src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="Surname" class="elem"> Surname:
+          <input
+            type="text"
+            name="Surename"
+            autocomplete="off"
+            required
+            minlength="5"
+            maxlength="13"
+          >
+        </label>
+        <label for="name" > Name:
+          <input
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label for="age" class="elem"> How old are You?
+          <input
+            type="number"
+            name="number"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+        <label for="datOfBirth" class="elem"> Full date of birth:
+          <input
```

Comment:
text inside tags should be placed beneath, mind it, please
```suggestion
        <label for="datOfBirth" class="elem">
          Full date of birth:
          <input
```